.travis.yml

```diff
@@ -7,8 +7,9 @@ php:
 
 env:
   - SULU_ORM=mysql SULU_PHPCR=jackrabbit
-#  - SULU_ORM=mysql SULU_PHPCR=doctrine_dbal
 #  - SULU_ORM=postgres SULU_PHPCR=jackrabbit
+#  - SULU_ORM=mysql SULU_PHPCR=doctrine_dbal
+#  - SULU_ORM=postgres SULU_PHPCR=doctrine_dbal
```

Comment:
Yeah, but you probably should uncomment the first line containing postgres, because the tests currently also don't run with a `SULU_PHPCR` value of `doctrine_dbal`